doc/install.md

```diff
@@ -1,39 +1,75 @@
+2. Untar them with `tar -xzf livepeer-linux-amd64.tar.gz` or `tar -xzf livepeer-darwin-amd64.tar.gz`
+
+### Running the software
+
+You can then run the `livepeer` binary with the following simple command.
+```
+./livepeer-linux-amd64/livepeer -broadcaster
+```
+This creates an `RTMP` endpoint on `127.0.0.1:1935` and an `HLS` media server on `127.0.0.1:8935`.
+
+### Basic test of the installation
+
+You can serve `RTMP` content into the endpoint using the following command:
+```
+ffmpeg -re -f lavfi -i testsrc=size=360x640:rate=30,format=yuv420p -f lavfi -i sine -threads 1 -c:v libx264 -b:v 4000000 -preset ultrafast -x264-params keyint=30 -strict -2 -c:a aac -f flv rtmp://127.0.0.1:1935/test_signal
+```
+You can query the `HLS` content coming from the media server using:
+```
+curl http://127.0.0.1:8935/stream/test_signal.m3u8
+```
+If you receive the following message, your basic `livepeer` node is running:
+```
+#EXTM3U
+#EXT-X-VERSION:3
+#EXT-X-STREAM-INF:PROGRAM-ID=0,BANDWIDTH=4000000,RESOLUTION=360x640
+test_signal/source.m3u8
+```
+
+For regular builds published by Livepeer's automated build process, see below.
 
-### Option 2: Build from source
-You can also build the executables from scratch.
+## Option 2: Build from source
 
-&ensp; 1\. If you have never set up your Go programming environment, do so according to Go's [Getting Started Guide](https://golang.org/doc/install).
+You can also build your own executables from source code.
 
-&ensp; 2\. You can fetch the code running `go get github.com/livepeer/go-livepeer/cmd/livepeer` in terminal.
+### Pre-requisites and Setup
 
-&ensp; 3\. Make sure you have the necessary libraries installed:
+&ensp; 1\. Install Go, using Go's [Getting Started Guide](https://golang.org/doc/install).
 
-* Linux: `apt-get update && apt-get -y install build-essential pkg-config autoconf gnutls-dev`
+&ensp; 2\. Make sure you have the necessary libraries installed:
+
+ * Linux: `apt-get update && apt-get -y install build-essential pkg-config autoconf gnutls-dev git`
 
  * OSX: `brew update && brew install pkg-config autoconf gnutls`
 
-&ensp; 4\. You need to install `ffmpeg` as a dependency.  Run `./install_ffmpeg.sh`.  This will install the dependencies in `~/compiled`.  You need to have `pkg-config` installed.
+&ensp; 3\. Fetch the code running `go get github.com/livepeer/go-livepeer/cmd/livepeer` in terminal. This will download the software to `/src/github.com/livepeer/go-livepeer` in your `go` install folder.
+
+&ensp; 4\. You need to install `FFmpeg` as a dependency.  Run `./install_ffmpeg.sh` from the `go-livepeer` folder.
+
+### Make the software
 
-&ensp; 5\. You can now run `HIGHEST_CHAIN_TAG=<CHAIN_NAME> PKG_CONFIG_PATH=~/compiled/lib/pkgconfig make` from the project root directory. To get latest version, `git pull` from the project root directory.
+&ensp; 5\. You can now run `HIGHEST_CHAIN_TAG=dev PKG_CONFIG_PATH=~/compiled/lib/pkgconfig make` from the project root directory.
```

Comment:
Technically, `dev` is the default so `HIGHEST_CHAIN_TAG` can be omitted for most builds if the intent is to run offchain.

Reply:
Understood, however why not encourage people to be explicit, if it helps them to format the command, which they can change later.

Propose to keep it as is.

Reply:
There should be a mention that there's no guarantee that Rinkeby / mainnet builds will be 100% compatible off master, outside of a release. That can be done alongside  the optionality of the HIGHEST_CHAIN_TAG builds. Eg:

> Default builds are development builds and will not run on mainnet or Rinkeby. For a development build, run ... <command line without HIGHEST_CHAIN_TAG>

> For a Rinkeby or mainnet build, the HIGHEST_CHAIN_TAG environment variable needs to be set. Use with caution: non-release builds are not guaranteed to be compatible with Rinkeby or mainnet. To make a  Rinkeby build, use `HIGHEST_CHAIN_TAG=rinkeby` . For a mainnet build, use `HIGHEST_CHAIN_TAG=mainnet`. For example: ...

Reply:
We [originally chose](https://github.com/livepeer/go-livepeer/pull/1261#discussion_r355776209) to be explicit about not omitting the `HIGHEST_CHAIN_TAG` env variable in the documentation here nor the command (for obvious reasons). 

So from my perspective there's a couple of things we want the user to understand here:
- What `HIGHEST_CHAIN_TAG` does
- What the available values are
- That in can be omitted in case of local network / offchain mode 

I can see that the original description might not be entirely clear so how about just rewriting that part and leaving the instruction itself as described in [this comment](https://github.com/livepeer/go-livepeer/pull/1261#discussion_r355776209).

I feel like the original text does a pretty decent job at balancing brevity and clarity. 

In order to tackle the last point how about we just add 

> This environment variable can be omitted when building for a local Ethereum network or an offchain environment.

Reply:
I think this is resolved here:

https://github.com/livepeer/go-livepeer/commit/303ef94b3a16e9aaafe76490e11c5e53535c6871

Reply:
I feel like it's not, the new description is less clear than the original one so my suggestion is to adopt the proposal above or revert the change.

Reply:
The new steps are, imo, simpler to follow - because:

1. they start by giving a very simple copy-and-paste `... make` command, without requiring understanding / editing of parameters - i.e. the call to action is simpler to perform.

2. they take into account @j0sh's:

> Technically, dev is the default so `HIGHEST_CHAIN_TAG` can be omitted for most builds if the intent is to run offchain.

If it's video developers that we want to engage, I would say that `-offchain` is the best place to start them, without the crypto complexity.

3. they do still introduce the `HIGHEST_CHAIN_TAG` in a subsequent step, with very explicit instructions for how it can be used, again with very simple copy-and-paste commands, no parameters / editing needed.

@kyriediculous here is the proposed text, for sake of clarity: https://github.com/criticaltv/go-livepeer/blob/patch-1/doc/install.md#make-the-software

@j0sh perhaps you can mediate?

Reply:
This looks good to me. It's often helpful to be more explicit than not.

Reply:
Wait, which one looks good to you?